src/button.js

```diff
@@ -158,12 +158,24 @@ export default class Button {
 			ok = options.click.call(this, event);
 		}
 		if (ok) {
+			if (this.widget.getAttribute('data-url') !== this.options.url) {
```

Comment:
Sorry, it's another topic to discuss, because sometimes for each social network we need different links with different UTM's. And now it's impossible. And I can't change this links on fly.